plugins/kotlin-dataframe/tests-gen/org/jetbrains/kotlin/fir/dataframe/DataFrameBlackBoxCodegenTestGenerated.java

```diff
@@ -233,6 +233,12 @@ public void testFlexibleReturnType() {
     runTest("testData/box/flexibleReturnType.kt");
   }
 
+  @Test
+  @TestMetadata("sum.kt")
```

Comment:
After you create new file in testData, please use this task to update
![image](https://github.com/user-attachments/assets/0db96ebf-e81b-4d9e-845c-8f38cf170497)

Reply:
Missed it, thanks